src/client/dfuse/il/int_posix.c

```diff
@@ -320,6 +378,15 @@ ioil_init(void)
 		return;
 
 	ioil_iog.iog_initialized = true;
+
+	rc = pthread_mutex_lock(&ioil_iog.iog_lock);
+	D_ASSERT(rc == 0);
+
+	if (!ioil_iog.iog_daos_init)
+		(void)call_daos_init(0);
```

Comment:
What would happen if this call failed?  I think it would be fine wouldn't it, the function cleans up after itself and there's no other state to unwind, and it wouldn't then get called a second time?

Reply:
yeah, that was my thinking

Reply:
The other place it's called doesn't print anything either when it fails.  I think any prints would be internal so there is really nothing to do with the return code.